src/HomeWork/NumbRevers/NumberRevers.java

```diff
@@ -0,0 +1,43 @@
+package HomeWork.NumbRevers;
+
+import java.util.Scanner;
+
+public class NumberRevers {
+    public static void main(String[] args) {
+        boolean isExit;
+        do {
+            try {
+                Scanner scanner = new Scanner(System.in);
+                System.out.print("Введіть кількість цифр яку реверснемо: ");
+                int count = scanner.nextInt();
+
+                int[] numbers = new int[count];
+                System.out.println("Введіть цифри:");
```

Comment:
it is a little bit complicated, i think it will be enough to ask a user to enter just a number (f.e 2376 or 87658) without splitting to digits. 
the logic of splitting that number should be handled in the code